pkg/hintrunner/hint.go

```diff
@@ -250,3 +250,161 @@ func (hint SquareRoot) Execute(vm *VM.VirtualMachine) error {
 	}
 	return nil
 }
+
+type Uint256SquareRoot struct {
+	valueLow                     ResOperander
+	valueHigh                    ResOperander
+	sqrt0                        CellRefer
+	sqrt1                        CellRefer
+	remainderLow                 CellRefer
+	remainderHigh                CellRefer
+	sqrtMul2MinusRemainderGeU128 CellRefer
+}
+
+func (hint Uint256SquareRoot) String() string {
+	return "Uint256SquareRoot"
+}
+
+func (hint Uint256SquareRoot) Execute(vm *VM.VirtualMachine) error {
+	valueLow, err := hint.valueLow.Resolve(vm)
+	if err != nil {
+		return fmt.Errorf("resolve valueLow operand %s: %v", hint.valueLow, err)
+	}
+
+	valueHigh, err := hint.valueHigh.Resolve(vm)
+	if err != nil {
+		return fmt.Errorf("resolve valueHigh operand %s: %v", hint.valueHigh, err)
+	}
+
+	valueLowFelt, err := valueLow.FieldElement()
+	if err != nil {
+		return err
+	}
+
+	valueHighFelt, err := valueHigh.FieldElement()
+	if err != nil {
+		return err
+	}
+
+	// value = {value_low} + {value_high} * 2**128
+	valueLowU256 := uint256.Int(valueLowFelt.Bits())
+	valueHighU256 := uint256.Int(valueHighFelt.Bits())
+	valueHighU256.Lsh(&valueHighU256, 128)
+	value := valueHighU256.Add(&valueHighU256, &valueLowU256)
+
+	// root = math.isqrt(value)
+	root := value.Clone()
+	root.Sqrt(value)
+
+	// remainder = value - root ** 2
+	root2 := root.Clone()
+	root2.Mul(root, root)
+	remainder := value.Clone()
```

Comment:
reamainder can be an empty value as well

Reply:
I'll stop signaling since the patter repeats further down below. It hurts readability initializing variables with values they are not going to use

Reply:
You should be able to do 
```
var root = uint256.Int{}
root.Sqrt(&value)
```
What memory errors are you getting?